bindings/capacitorjs/ios/Podfile

```diff
@@ -0,0 +1,42 @@
+platform :ios, '12.0'
+
+def capacitor_pods
+  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
+  use_frameworks!
+  pod 'Capacitor', :path => '../node_modules/@capacitor/ios'
+  pod 'CapacitorCordova', :path => '../node_modules/@capacitor/ios'
+  # pod 'IotaWalletInternal', :podspec => './packages/mobile/capacitor/plugins/iota-wallet/IotaWalletInternal.podspec'
+  
+end
+
+target 'Plugin' do
+  capacitor_pods
+  # use_frameworks!
+  # pod 'IotaWalletInternal', :path => 'WalletFramework.xcframework/'
+end
+
+target 'PluginTests' do
+  capacitor_pods
+end
+
+# post_install do |installer|
+#   installer.pods_project.build_configurations.each do |config|
+#     config.build_settings["EXCLUDED_ARCHS[sdk=iphonesimulator*]"] = "arm64"
+#   end
+
+#   # Add WalletFramework.xcframework as a dependency of IotaWalletMobile
+#   # Based on https://github.com/CocoaPods/Xcodeproj/issues/408#issuecomment-249347379
+#   # and https://www.rubydoc.info/github/CocoaPods/Xcodeproj/Xcodeproj/Project
+
+#   # Get the PBXFileReference to the framework
+#   pods = installer.pods_project.groups.find { |group| group.name == 'Pods' }
+#   pod = pods.children.find { |child| child.name == 'ios/WalletFramework.xcframework' }
+#   frameworks = pod.children.find { |child| child.name == 'Frameworks' }
+#   framework_file_ref = frameworks.children.first
+
+#   target = installer.pods_project.targets.find { |target| target.name == 'IotaWalletMobile' }
+
+#   # Add the framework to "Link Binary with Libraries"
+#   frameworks_build_phase = target.build_phases.find { |build_phase| build_phase.to_s == 'FrameworksBuildPhase' }
+#   frameworks_build_phase.add_file_reference(framework_file_ref)
+# end
```

Comment:
What's up with all the commented lines?

Reply:
this dead code is already deleted in the original PR